portal/config/eproms/Questionnaire.json

```diff
@@ -6965,6 +6965,15 @@
         {
           "type": "choice",
           "linkId": "ironman_ss.1",
+          "code":  {
+            "coding": [
```

Comment:
I don't think we want a `coding` wrapping the code here, at least going from the STU3 Questionnaire examples (another [here with a populated](http://hl7.org/implement/standards/fhir/STU3/questionnaire-zika-virus-exposure-assessment.json) `Questionnaire.item.code`)

Reply:
@ivan-c  done, see my latest commit here.